server/routes/profile.js

```diff
@@ -36,6 +37,7 @@ router
 
 router.route("/edit").put(protect, editProfile);
 
+router.route("/load/:id").get(loadProfileById);
 router.route("/load").get(protect, loadProfile);
```

Comment:
I can see these routes were defined prior to this PR, but I think this is an appropraite place to adjust them to better suit REST resource naming conventions - as mentioned [here](https://restfulapi.net/resource-naming/): 
>## 2.4. Never use CRUD function names in URIs
>We should not use URIs to indicate a CRUD function. URIs should only be used to uniquely identify the resources and not any action upon them.
>
>We should use HTTP request methods to indicate which CRUD function is performed.

So I would recommend getting rid of `edit/load` words, the routes would still remain distinguishable by their HTTP verb 👍🏼